tests/test_medline_parser.py

```diff
@@ -1,9 +1,157 @@
 import os
 import pytest
+import requests
+import gzip
+from io import BytesIO
 
 import pubmed_parser as pp
 from pubmed_parser import split_mesh
 
+def fetch_compressed_medline_xml(pubmed_id):
+    """Fetch up-to-date pubmed XML and return as compressed XML"""
+    url = f"https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pubmed&retmode=xml&id={pubmed_id}"
+    response = requests.get(url)
+    if response.status_code == 200:
+        compressed_stream = gzip.compress(response.content)
+        compressed_stream = BytesIO(compressed_stream)
+        return compressed_stream
+    else:
+        raise requests.exceptions.HTTPError(response.status_code)
+
+
+parsed_medline = pp.parse_medline_xml(fetch_compressed_medline_xml(['36400559', '28786991']))
+parsed_medline = list(parsed_medline)
+article_36400559 = parsed_medline[0]
+article_28786991 = parsed_medline[1]
+
+def test_medline_abstract():
```

Comment:
This an all other functions: Drop the "medline_": Literally just use the property name.